proj/src/main/webapp/index.html

```diff
@@ -27,6 +27,7 @@
 
         <script src="https://www.gstatic.com/firebasejs/7.15.4/firebase-database.js"></script>
         <script src="https://www.gstatic.com/firebasejs/7.15.4/firebase-messaging.js"></script>
+        <script src="app.js"></script> <!-- i put the initialization stuff in here so i don't have to take it out the commits every time -->
```

Comment:
I think the comment can be done in the PR rather than in the file. Or maybe in the file just have something like `<!--Holds initialization scripts -->`

Reply:
fixed!